R/reportMif.R

```diff
@@ -53,7 +54,12 @@ reportMif <- function(path, file = NULL, tmpl = NULL) {
 
   # reporting template
   if (is.null(tmpl)) {
-    tmpl <- config[["reportingTemplate"]]
+    tmplFile <- file.path(path, "config", "brickSet_COMPILED.yaml")
```

Comment:
This should be ```brickSets_COMPILED.yaml``` because that is the name the file is written to in ```createRunFolder```.

Reply:
Thanks, well spotted!